app/repositories/witter/friki_phrase.js

```diff
@@ -0,0 +1,11 @@
+const axios = require('axios');
+
+const API_URL = 'https://geek-jokes.sameerkumar.website/api?format=json';
```

Comment:
this should be an environment variable. Look `config/index.js`

Reply:
Thanks, in the last commit I made the change to be retrieved from an environment variable.